docs/output.md

```diff
@@ -1,50 +1,64 @@
-# nf-core/cutandrun: Output
+<!-- vscode-markdown-toc -->
+
+- 1. [Introduction](#Introduction)
```

Comment:
ok, so the links here:
https://nf-co.re/cutandrun/dev/output#Introduction
and the one automatically made by the MD:
https://nf-co.re/cutandrun/dev/output#1--introduction
they don't appears to resolve to exactly the same location.

@mashehu any ideas/comments?